includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -318,106 +318,6 @@ public function clear_fields() {
 		$this->fields = [];
 	}
 
-	/**
-	 * Enqueue the scripts for the form.
-	 */
-	public function enqueue_scripts() {
```

Comment:
As these are public functions, we will need to deprecate themn for a couple of versions until we eventually remove them as they will break sites that use them. So we should basically add `deprecated_function` calls and call the function from the recaptcha class.

We use this method in applications also which causes this fatal:
```
PHP Fatal error:  Uncaught Error: Call to undefined method WP_Job_Manager_Form::enqueue_scripts() in /wp-content/plugins/wpjm-addons/src/applications/includes/class-wp-job-manager-applications-apply.php:79
Stack trace:
```

Reply:
Deprecated in 74ce7d0c5322d1e6579d44e13bbf999e13e9c0b6